src/services/schemas/properties/SchemaPropertyAny.ts

```diff
@@ -8,19 +8,19 @@ import { parseUnknownJson, stringifyUnknownJson } from '@/utilities/json'
 
 export class SchemaPropertyAny extends SchemaPropertyService {
   protected get default(): unknown {
+    let defaultValue: unknown
+
     if (this.has('default')) {
-      return getSchemaPropertyResponseValue(this.property, this.property.default, this.level + 1)
+      defaultValue = this.property.default
+    } else if (this.has('anyOf') || this.has('allOf')) {
+      defaultValue = this.getDefaultValueForFirstDefinition()
     }
 
     if (this.componentIs(JsonInput)) {
-      return ''
-    }
-
-    if (this.has('anyOf') || this.has('allOf')) {
-      return this.getDefaultValueForFirstDefinition()
+      return stringifyUnknownJson(defaultValue) ?? ''
     }
 
-    return null
+    return defaultValue ?? {}
```

Comment:
Changing the fallback value (if there is no default value) from `null` to `{}` seems risky to me. Is returning an empty object specifically needed?

Reply:
Hm shouldn't be, no - reverted